test-utils/src/main/kotlin/dev/restate/e2e/utils/annotations.kt

```diff
@@ -15,6 +15,8 @@ internal const val RUNTIME_META_ENDPOINT_PORT = 9070
 
 @Target(AnnotationTarget.VALUE_PARAMETER) annotation class InjectBlockingStub
 
+@Target(AnnotationTarget.VALUE_PARAMETER) annotation class InjectFutureStub
```

Comment:
You've introduced this but you're not using it in the next commit right? Also the future stubs are not needed with kotlin, because we have the coroutine stubs already being generated. We just have never used them. For a usage example see https://github.com/restatedev/examples/blob/main/kotlin/hello-world-http/src/test/kotlin/dev/restate/sdk/examples/GreeterTest.kt#L25

Reply:
Yes, it is not needed. I will remove it.